### PR TITLE
feat: Support for single file sync processing

### LINKED ISSUE #84: Ability to run for a single file, and return the modified content as a string
That way we could use tsc-alias as a custom jest transformer.

### PR DESCRIPTION
**Pull request by eturino**
- adding `prepareSingleFileReplaceTscAliasPaths(options?)`, which will return a promise of the singleFileReplacer function, with the config already prepared.
- `SingleFileReplacer` is a function that will receive `{fileContents: string, filePath: string}` and return the treated contents

fix #84

### CODE REVIEW

File: tests/test.spec.ts [hunk 1]
```diff
@@ -90,3 +97,56 @@ it(`Import regex does not match edge cases from keywords in strings`, function (
...
+    `!${basePath}/**/node_modules`
+  ];
+  const files = sync(globPattern, {
+    dot: true,
+    onlyFiles: true
+  });
+
+  console.log(
+    JSON.stringify({ files, outPath, basePath, options, config }, null, 2)
+  );
```
Comment (eturino) [inline tests/test.spec.ts:139]: leftovers
```suggestion
```

File: tests/test.spec.ts [hunk 2]
```diff
@@ -90,3 +97,53 @@ it(`Import regex does not match edge cases from keywords in strings`, function (
...
+  const globPattern = [
+    `${basePath}/**/*.{mjs,cjs,js,jsx,d.{mts,cts,ts,tsx}}`,
+    `!${basePath}/**/node_modules`
+  ];
+  const files = sync(globPattern, {
+    dot: true,
+    onlyFiles: true
+  });
+
+
```
Comment (eturino) [inline tests/test.spec.ts:137]:
```suggestion
```
